docs/1-mcp.md

```diff
@@ -149,13 +149,13 @@ Now that you have set up the GitHub MCP server, you can use Copilot Agent mode t
 
     ![Example of the Copilot Chat pane with Agent Mode selected](images/copilot-agent-mode-dropdown.png)
 
-2. Type or paste the following prompt. Note that you will need to replace **micro/repository** with the name of your GitHub repository, e.g. Microsoft-Build-2025/your-repo-name:
+2. Type or paste the following prompt to create a the issues we'll be working on in the lab:
 
 ```markdown
-Create GitHub issues for our Tailspin Toys backlog in the **organization-name/repository** repo.  Each issue should include:
+Create GitHub issues for our Tailspin Toys backlog in my repo. Each issue should include:
 - A clear title
 - A brief description of the task and why it is important to the project
-- A checkbox list of acceptance criteria / definition of done
+- A checkbox list of acceptance criteria
```

Comment:
We'll need to revert back to the previous version, and have the user type in the repository name. Using "my repository" is not robust enough, as the agent doesn't check for the org owner.

My suggestion is the following:

2. Type or paste the following prompt. Note that you will need to replace **organization/repository** with the name of your GitHub repository, e.g. Microsoft-Build-2025/310-geektrainer:

 ```markdown
 Create GitHub issues for our Tailspin Toys backlog in the **organization/repository** repo.  Each issue should include:
```